### DISCUSSION TITLE
November Newsletter!

### DISCUSSION THREAD

**Discussion by rsshaik** [usememos/memos#4158] [General]
Here’s your November newsletter, packed with updates, tips, and everything you need to make the most of Memos. Let’s dive in! 

### Features You Might Have Missed

  📋 **Embedded Content**
  You can embed content into your memos for cross-referencing. Use [[memos/memoid]] to link directly to another memo. Perfect for creating interconnected notes!

  🌐 **HTML Rendering**
  Memos supports rendering HTML directly in your memos. Syntax for embedded content and html code blocks can be found [here](https://www.usememos.com/docs/getting-started/content-syntax) to create more visually appealing content.

  🎯 **Customizable Navigation**
  The new collapsible navigator lets you hide sections you don’t need, keeping your workspace neat and focused.

  🔗 **Tagging**
  Tags are more powerful than they seem! Add #tags at the start or end of your memos to group and filter content effortlessly.
    ✅ Best Practice: Add tags like #ideas #tasks at the start or end of your content for accurate parsing.
    🚫 Avoid: Placing tags in the middle of your content as they might not get recognized properly.

  📊 **Track Memos with Short UUIDs**
  Memos now uses short UUIDs for identifying content. This makes URLs and IDs cleaner and easier to manage, especially in shared environments.

### Here’s a quick refresher on deploying Memos with Docker for efficiency:
`docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:stable`

### 🌟 Creative Ways to Use Memos
Here are some creative ways users are leveraging Memos:
  👩‍💻 **Organizing thoughts:** Organize learning resources, tutorials, and articles for quick access.
  📚 **Collaborative Workspaces:** Use Memos for team projects with shared tagging and referencing.
  📝 **Journaling:** Track your thoughts and goals with private, time-stamped entries.

### 🚀 Stay Up-to Date
The latest release, **[v0.23.0-rc.2](https://github.com/usememos/memos/releases/tag/v0.23.0-rc.2)**, is packed with the newest features and bug fixes. Update now to enhance your experience!

### Links you might want to check out

- Best practices to write a TAG https://www.usememos.com/blog/best-practices-to-write-tag

- Released v0.22.4  https://github.com/usememos/memos/releases (previous version)

### 💬 Join the Conversation
We love hearing from you! Got feedback, ideas, or suggestions? Please let us know here!

**Comment by RoccoSmit** [comment]
Nice one @rsshaik ! Thanks for putting this together

Couple of notes:
- syntax for `embedded content` and `html` code blocks [here](https://www.usememos.com/docs/getting-started/content-syntax) 
- due to unfortunate timing (it came out today as well) the latest release is [v0.23.0-rc.2](https://github.com/usememos/memos/releases/tag/v0.23.0-rc.2)
- v0.19.0 is somewhat outdated, v0.22.4 was the last release with feature documentation https://github.com/usememos/memos/releases

**Reply by rsshaik** [reply]
Thank you @RoccoSmit ! I made the necessary changes to the post.
